x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/ner/ner_output.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import React, { FC, ReactNode } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiHorizontalRule,
+  EuiBadge,
+  EuiToolTip,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+} from '@elastic/eui';
+
+import type { FormattedNerResp } from './ner_inference';
+
+export const NerOutput: FC<{ result: FormattedNerResp }> = ({ result }) => {
+  const lineSplit: JSX.Element[] = [];
+  result.forEach(({ value, entity }) => {
+    if (entity === null) {
+      const lines = value
+        .split(/(\n)/)
+        .map((line) => (line === '\n' ? <br /> : <span>{line}</span>));
+
+      lineSplit.push(...lines);
+    } else {
+      lineSplit.push(
+        <EuiToolTip
+          position="top"
+          content={
+            <div>
+              <div>
+                <EuiIcon
+                  size="m"
+                  type={getClassIcon(entity.class_name)}
+                  style={{ marginRight: '2px', verticalAlign: 'text-top' }}
+                />
+                {value}
+              </div>
+              <EuiHorizontalRule margin="none" />
+              <div style={{ fontSize: '12px', marginTop: '2px' }}>
```

Comment:
we should use EUI theme params for the font size

Reply:
Updated in https://github.com/elastic/kibana/pull/128359/commits/2a9f47840a7e077a838e19e46673fad819b0f80c